api.ts

```diff
@@ -1035,11 +1035,21 @@ export interface SyncFormulaResult<K extends string, L extends string, SchemaT e
 
   /**
    * Return the list of deleted item ids for incremental sync deletion.
-   * 
+   *
    * TODO(ebo): Unhide this
    * @hidden
    */
   deletedItemIds?: string[];
+
+  /**
+   * Returned by an incremental sync if a full sync is needed to recalculate the items.
+   *
+   * This will trigger a full sync so it should be avoided when possible
+   *
+   * TODO(sam): Unhide this
+   * @hidden
+   */
+  triggerFullSync?: boolean;
```

Comment:
Could also call this something like "has incomplete results" if we want to stay away from this language

Reply:
Hmm, yeah generally we use declarative framings rather than imperative framings. It's up to the system to decide what to do with the information you give it, you don't really give commands. I think it's fine if the documentation says that in practice returning this will trigger a full sync but ultimately you're just describing the result set and it's up to Coda to figure out how to respond to that.

Reply:
I feel like this might most "appropriately" live under `completion`? cc @ekoleda-codaio

Reply:
I'm fine with the name as-is.

Reply:
I like completion better as a home, but would that mean storing the incomplete status in the continuation until you finish the incremental sync and issue the completion?